test/jasmine/tests/parcoords_test.js

```diff
@@ -219,7 +219,7 @@ describe('parcoords initialization tests', function() {
     });
 });
 
-describe('parcoords', function() {
+describe('@noCI parcoords', function() {
```

Comment:
@monfera this suite here blows the process on CI.

Maybe next time you play around with the `parcoords` code you could try to investigate which `it`(s) exactly blow(s) up the process. Or better yet, try to make the test less resource intensive. Thanks in advance!